src/Microsoft.MobileBlazorBindings/Elements/AttributeHelper.FlowDirection.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace Microsoft.MobileBlazorBindings.Elements
+{
+    public static partial class AttributeHelper
```

Comment:
Enum values don't need helpers because enums are trivially converted to/from ints.

For example, check out `StackLayout.Orientation`:

Component code: https://github.com/xamarin/MobileBlazorBindings/blob/master/src/Microsoft.MobileBlazorBindings/Elements/StackLayout.cs#L31-L34
Handler code: https://github.com/xamarin/MobileBlazorBindings/blob/master/src/Microsoft.MobileBlazorBindings/Elements/Handlers/StackLayoutHandler.cs#L22-L24

Also, I'm trying to keep the properties sorted alphabetically both where they're declared and also where they're rendered/processed. This will make it easier to update things with tooling.

Reply:
> Enum values don't need helpers because enums are trivially converted to/from int
Totally agree, I'm gonna change it and thanks for some examples.

> Also, I'm trying to keep the properties sorted alphabetically both where they're declared and also where they're rendered/processed. This will make it easier to update things with tooling.
Interesting.

Reply:
I removed the helper for `FlowDirection` helper, changed the code as you mentioned and pushed the changes.